### PR TITLE
Setting up file structure: Item class

### PR DESCRIPTION
**Pull request by Anny85-code**
### In this milestone we ;
- [x] Created the item class.
- [x] Created the methods namely:
- author
- label
- genre
- can_be_archived
- move_to_archived.

### CODE REVIEW

File: classes/item.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  def genre=(genre)
+    @genre = genre
+    genre.items.push(self) unless genre.items.include?(self)
+  end
+
+  private :can_be_archived?
+
+  def can_be_archived?
+    return true if @publish_date < DateTime.now.next_year(-10)
+
```
Comment (orozCoding) [inline classes/item.rb:31]: I suggest that the `private...` line goes after the can_be_archived? method definition 👍

**Review by orozCoding — CHANGES_REQUESTED** [review]
# Great job!

I believe we almost finished. There's a small issue with the can_be_archived? method. Please read the comments bellow for more details

File: classes/item.rb (hunk 1 above)
Comment (Anny85-code) [reply classes/item.rb]: Thank you for your review, changes implemented.

**Review by orozCoding — APPROVED** [review]
# APPROVED!

Everything is working fine. We are ready to move on

**Pull request merged**